inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_mmad_bfyx_to_b_fs_yx_fsv32.cl

```diff
@@ -239,17 +239,34 @@ KERNEL(convolution_mmad_bfyx_to_b_fs_yx_fsv32)(
         if (z_cross_fm)
             continue;
 #endif
+        PACKED_IN_TYPE line_cache[INPUT_LINE_SIZE * OUTPUT_Y_BLOCK_SIZE];
         __attribute__((opencl_unroll_hint(FILTER_SIZE_Y)))
         for (int kh = 0; kh < FILTER_SIZE_Y ; ++kh) {
+#if OUTPUT_Y_BLOCK_SIZE > 1
+            for (int yb = 0; yb < OUTPUT_Y_BLOCK_SIZE; yb++) {
+                bool y_cross_fm = input_y + ((kh*DILATION_SIZE_Y)+yb*STRIDE_SIZE_Y) < 0 || input_y + ((kh*DILATION_SIZE_Y)+yb*STRIDE_SIZE_Y) >= INPUT0_SIZE_Y;
```

Comment:
I think you should replace this line under define`ASYMMETRIC_DATA_QUANTIZATION`. And the same in the code line 261.